worker_test.go

```diff
@@ -322,18 +589,17 @@ func TestResourceConversion(tt *testing.T) {
 				scenario.typeLB,
 				clusterLocalDomain,
 				true)
-			if err != nil {
-				t.Fatal(err)
-			}
+			require.NoError(t, err)
+
 			log.SetLevel(log.DebugLevel)
 			problems := doWork(&mockedCertificateProvider{t: t}, 10, time.Hour, a, k, "")
-			if len(problems.Errors()) > 0 {
-				t.Error(problems.Errors())
-			}
+			require.Empty(t, problems)
+
+			// TODO: assert on number of stacks created (currently 1) to check non-shared load balancer
```

Comment:
Non Shared create 2 stacks which could be known from metrics, but this require some changes to metrics, maybe we can address it in another PR.